saleor/graphql/schema.graphql

```diff
@@ -21781,6 +21797,53 @@ type OrderLineDiscountRemove @doc(category: "Orders") {
   errors: [OrderError!]!
 }
 
+"""
+Adds note to the order. 
+
+Requires one of the following permissions: MANAGE_ORDERS.
+"""
+type OrderNoteAdd @doc(category: "Orders") {
+  """Order with the note added."""
+  order: Order
+
+  """Order note created."""
+  event: OrderEvent
+  orderErrors: [OrderError!]! @deprecated(reason: "This field will be removed in Saleor 4.0. Use `errors` field instead.")
+  errors: [OrderError!]!
+}
+
+input OrderNoteInput @doc(category: "Orders") {
+  """Note message."""
+  message: String!
+}
+
+"""
+Removes note from an order. 
+
+Requires one of the following permissions: MANAGE_ORDERS.
+"""
+type OrderNoteRemove @doc(category: "Orders") {
```

Comment:
Why this mutation doesn't return OrderEvent for action? 
We have Order Event in `OrderNoteUpdate` and `OrderNoteAdd`.